src/main.ts

```diff
@@ -341,14 +345,22 @@ async function getAllActionsFromForkedRepos(
         actionFiles.length - 1
       }] action in repo [${repoName}] that was cloned to [${repoPath}]`
     )
+
     for (let index = 0; index < actionFiles.length - 1; index++) {
       core.debug(`Found action file [${actionFiles[index]}] in repo [${repoName}]`)
 
       // remove the actions/$repopath
       const actionFile = actionFiles[index].substring(`actions/${repoName}/`.length)
       core.debug(`Found action file [${actionFile}] in repo [${repoName}]`)
+      
+      // Get Repository Details
+      const repoDetail = await getRepoDetails(client, repoOwner, repoName)
+      const isArchived = repoDetail.archived
+
```

Comment:
This extra call is not needed: in the repo search the `archived` field is available. Not making this call will save us unnecessary rate limit waits.

![image](https://github.com/devops-actions/load-available-actions/assets/6085745/7a6a9368-2e33-4db7-9cc0-e81ba2469c59)

Reply:
Extra call removed and using the repo form the search results, indeed the archived field is there.